productcomments.php

```diff
@@ -896,6 +896,30 @@ public function hookDisplayFooterProduct($params)
         return $this->renderProductCommentsList($params['product']) . $this->renderProductCommentModal($params['product']);
     }
 
+    /**
+     * Inject data about productcomments in the product object for frontoffice
+     *
+     * @param array $params
+     *
+     * @return void
+     */
+    public function hookFilterProductContent(array $params)
+    {
+        if (empty($params['object']) || empty($params['object']->id)) {
+            return;
+        }
+        /** @var ProductCommentRepository $productCommentRepository */
+        $productCommentRepository = $this->context->controller->getContainer()->get('product_comment_repository');
+
+        $averageRating = $productCommentRepository->getAverageGrade($params['object']->id, (bool) Configuration::get('PRODUCT_COMMENTS_MODERATE'));
+        $commentsNb = $productCommentRepository->getCommentsNumber($params['object']->id, (bool) Configuration::get('PRODUCT_COMMENTS_MODERATE'));
+
+        $params['object']->productComments = [
+            'averageRating' => $averageRating,
+            'numComments' => $commentsNb,
```

Comment:
```suggestion
            'nbComments' => $commentsNb,
```

Reply:
why ?

Reply:
nb stands for number, right?

so, it's:

`commentsNb` or `nbComments` :D

Reply:
Fixed